model/minio/minio_cluster.rb

```diff
@@ -60,4 +67,29 @@ def single_instance_multi_drive?
   def hostname
     "#{name}.#{Config.minio_host_name}"
   end
+
+  def certs
+    root_cert_1 + root_cert_2
+  end
+
+  def run(access_key, secret_key, endpoint: nil, socket_path: nil, cmd: :cmd, **args)
+    ca_path = File.join(Dir.pwd, "ca_bundles", ubid.to_s)
+    FileUtils.mkdir_p(ca_path)
+    File.write(File.join(ca_path, "ca_bck.pem"), certs)
+    File.rename(File.join(ca_path, "ca_bck.pem"), File.join(ca_path, "ca.pem"))
```

Comment:
This is safe against crashes in midway, thanks to use of File.rename, but is not safe against concurrent updates of ca_bck.pem file.You need something like [this](https://github.com/ubicloud/ubicloud/blob/main/rhizome/common/lib/util.rb#L60)

Also maybe rename as ca.tmp? bck implies that is is backup.

Reply:
refactored as requested.